test/command.bats

```diff
@@ -23,3 +23,18 @@ load helpers
 	# can't cover everything here, ulimits parsing is tested in
 	# github.com/docker/go-units package
 }
+
+@test "invalid devices" {
+	run ${CRIO_BINARY} --additional-devices /dev/sda:/dev/foo:123
```

Comment:
it's gonna be tough to test this actually :thinking: maybe we need to create a loopback device on tests startup and try to mount that

Reply:
that could be one reliable way to have a known device on the platform, @mrunalp wdyt?